akka-typed-testkit/src/test/scala/akka/typed/testkit/EffectfulActorContextSpec.scala

```diff
@@ -27,11 +34,31 @@ object EffectfulActorContextSpec {
             ctx.spawn(Child.initial, s"child$i")
           }
           Actor.same
+        case SpawnChildrenWithProps(numberOfChildren, props) if numberOfChildren > 0 ⇒
```

Comment:
Now this dummy Behavior, can `spawn`, `spawnAnonymous`, with and without `Props`. And can `spawnAdapter` with and without name. These cases were added to cover all possible combinations.